lib/cmd/fh3/notifications/list.js

```diff
@@ -0,0 +1,50 @@
+/* globals i18n */
+var common = require('../../../common.js');
+var fhreq = require("../../../utils/request");
+var util = require('util');
+
+module.exports = {
+  'desc' : i18n._('List notifications of a RHMAP App'),
+  'examples' : [{
+    cmd : 'fhc notifications list --app=<app> --env=<environment> --audit=<true>',
```

Comment:
`--audit=true` has shorthand as `--audit` 
Worth reading yargs documentation - they have a lots of command formats.